PersistentVolume/README.md

```diff
@@ -2,3 +2,18 @@ Persistent volumes are not covered here as they require third party dependencies
 
 https://kubernetes.io/docs/concepts/storage/persistent-volumes/
 https://kubernetes.io/docs/concepts/storage/volume-snapshots/
+
+You can test a local `PersistentVolume` and a `PersistentVolumeClaim` with the following files. (this was tested on a docker-desktop kubernetes cluster)
```

Comment:
I think I'd prefer this to be in a separate PersistentVolumeClaim folder rather than mixed in with the PersistentVolume.